slides/week7/week7.org

```diff
@@ -0,0 +1,55 @@
+#+TITLE: Week 7
+#+AUTHOR: Josh Ting 
+#+EMAIL: josh.ting@gatech.edu
+
+* Data Structures
+- Data structures are ways that computers store collections of data in memory
+- Different data structures offer different advantages and disadvantages in terms of how data is grouped or how efficient it is to interact with the data
+
+* Arrays
+- You already have worked with arrays, which are the built in data structure for C++
+- Elements of arrays are ordered and easily retrievable
+- Arrays have a fixed size; maximum number of elements they can contain are determined when they are instantiated
+
+* Lists
+- A list is any data structure that is ordered
+- Useful when relative ordering between elements is important
+- There are various different implementations of lists in the C++ standard library
+
+* Linked Lists
+- Lists in which each data element is linked to the next element in the list
+- The programmer has access to the list's "head" (the first element) and sometimes the "tail" (the last element)
```

Comment:
This wording is a little bit wierd, it implies that we can't access anything but the head. Maybe say 'quick' or 'direct' access to the head/tail.